### ISSUE TITLE
[Feature Request] add prefix to grid classes

### ISSUE THREAD

**Issue by daskyrk** [vuetifyjs/vuetify#7039]
### Problem to solve
If implement the vuetify to an existing project, it's easy to break the original style for vuetify have a set of util style classes, like `container`, `layout` or `success` etc. And it also apply some style to default element like padding in `ul > li`, which could affect the existing style.

see this problem: https://stackoverflow.com/questions/54293336/how-to-avoid-vuetify-overrides-default-css

### Proposed solution
Maybe add a prefix to these util classes and not apply style to default element?

**Comment by johnleider** [comment]
I can see this happening for grid but not utilities.

**Comment by daskyrk** [comment]
in the cdn file: https://cdn.jsdelivr.net/npm/vuetify/dist/vuetify.min.css, you can format the file with chrome devtool, and there is some style applied to normal element like this:
![demo](https://ws3.sinaimg.cn/large/006tNc79ly1g2g1f8vp6ej30oa0iyjui.jpg)
and some other style starts from line 2734

By the way, class like 'container' is too normal so it's easy to conflict with exist style in project (:

**Comment by ipluser** [comment]
@johnleider 

I develop a Google Extension using Vuetify in content script, but it will overwrite the style of the original content page, then I append namespace class name for vuetify css, like below code:

```css
.custom-container {
  @import 'vuetify.min';
}
```

Next, I try to reduce the bundle size using [A-la-carte](https://vuetifyjs.com/en/framework/a-la-carte), like this:

```js
import Vuetify, {
  VApp,
  VNavigationDrawer,
  VFooter,
  VToolbar,
  VFadeTransition
} from 'vuetify/lib'

Vue.use(Vuetify, {
  components: {
    VApp,
    VNavigationDrawer,
    VFooter,
    VToolbar,
    VFadeTransition
  },
  // ...
})
```

But, It will inject the component style into content page using `<style></style>` tag, you know it will overwrite the original page~

I want to append prefix class name for all vuetify style, do you have some better solution? Thx~

**Comment by KaelWD** [comment]
@ipluser https://github.com/vuetifyjs/vuetify/issues/3706

**Comment by Dinver** [comment]
The problem is relevant. In the transition from 1.5 to 2.2. In Vuetify there should be no global styles. Styles should apply only to Vuetify components.

**Comment by josvos** [comment]
Just started evaluating Vuetify to see if I can / want to use it in my project and this is the first issue I came across in the first hour: the conflict with my existing container class, adding unwanted padding. It would be great if these generic classes can be renamed with a `v-` prefix.

**Comment by kosmall** [comment]
is any workaround for this moment?

**Comment by garbinmarcelo** [comment]
Facing the same problem with a project made with tailwindcss.

Any news about this option to add prefix in Vuetify?

**Comment by KaelWD** [comment]
This was done for grid classes in #12822

**Comment by johnleider** [comment]
resolved by #12822
